sign/impl.go

```diff
@@ -140,10 +182,17 @@ func (*defaultImpl) ParseReference(
 	return name.ParseReference(s, opts...)
 }
 
+// Deprecated: Please use FindTlogEntry
 func (d *defaultImpl) FindTLogEntriesByPayload(
-	ctx context.Context, rClient *client.Rekor, blobBytes []byte,
-) ([]string, error) {
-	return cosign.FindTLogEntriesByPayload(ctx, rClient, blobBytes)
+	ctx context.Context, rClient *client.Rekor, b64Sig string, blobBytes []byte, pubKey []byte,
+) ([]models.LogEntryAnon, error) {
+	return d.FindTlogEntry(ctx, rClient, b64Sig, blobBytes, pubKey)
+}
```

Comment:
Since this a method of the internal implementation, can we just drop it?

Reply:
done